dist/assembly.xml

```diff
@@ -141,6 +141,7 @@
       <includes>
         <include>org.hawkular.agent:hawkular-monitor:zip:module</include>
         <include>org.hawkular.bus:hawkular-bus-ra-wf-module:zip:module</include>
+        <include>org.hawkular.bus:cassandra-driver-wf-module:zip:module</include>
```

Comment:
Le 29/10/2015 10:12, Juraci Paixão Kröhling a écrit :
> It's in the hawkular-nest-distro, which doesn't seem to be consumed by
> Hawkular. Hawkular consumes the individual pieces of nest.

That's a surprise to me. Maybe @mazz knows why?